emcc.py

```diff
@@ -2313,6 +2313,14 @@ def parse_args(newargs):
         exit_with_error('Invalid value "' + newargs[i + 1] + '" to --output_eol!')
       newargs[i] = ''
       newargs[i + 1] = ''
+    elif newargs[i] == '--generate-config':
+      arg = newargs[i + 1];
```

Comment:
On CI, flake8 throws an error on this, for 2 reasons: the semicolon is not needed, and also it doesn't like that `arg` is used again later down. I'm not sure why the second is an error? anyhow, if you can't figure that out either, then please go back to the form before, with multiple uses of `newargs[i + 1]` instead of saving to a temp var

Reply:
i changed the temp variable's name as it is recommended here
https://lintlyci.github.io/Flake8Rules/rules/F812.html